platform/features/table/res/templates/scrolling.html

```diff
@@ -0,0 +1,9 @@
+<div ng-controller="RTTelemetryTableController">
+    <mct-table
+        headers="headers"
+        rows="rows"
+        enableFilter="true"
+        enableSort="true"
+        auto-scroll="true">
```

Comment:
Quite right, shouldn't be there. Previously had a third table type for message types that auto-scrolls. Folded into rt-table, but didn't discard the template.